README.md

```diff
@@ -1,2 +1,11 @@
-"# hibernate-hw" 
-"# hibernate-hw" 
+*** Cinema Project ***
+
+Technologies implemented: 
```

Comment:
Readme it's a must have for this project, but it makes sense to write it fully after you've competed your cinema completely, let's remove it from this PR